### DISCUSSION TITLE
CockroachDB out of Beta?

### DISCUSSION THREAD

**Discussion by danthegoodman1** [ory/kratos#1634] [Q&A] [answered]
Is there any timeline or predicted date for when CRDB support will no longer be considered beta?

**Comment by zepatrik** [comment, marked as answer]
Cockroach support is not considered beta, where did you read that? It is fully supported and works without a problem :wink:

**Reply by danthegoodman1** [reply]
@zepatrik 
https://www.ory.sh/docs/ecosystem/deployment/#cockroachdb-beta

But if you use the docs search bar for "cockroach" everything mentions beta.

**Reply by zepatrik** [reply]
Ok thanks, @aeneasr will have to confirm if that is up-to-date. To my knowledge it is not in beta anymore.

**Reply by danthegoodman1** [reply]
Yeah basically this and #655 are the only things I am waiting on before using Ory

**Reply by aeneasr** [reply]
We use CRDB in prod so it should be pretty stable :) We should update the docs there
